dart/lib/async/html.dart

```diff
@@ -25,6 +25,8 @@ export 'src/interfaces.dart';
 
 typedef Future SyncActionFn();
 
+Future _microtask(fn()) => new Future.microtask(fn).whenComplete(() {});
```

Comment:
Why is the "whenComplete" needed?

Maybe add a short comment explaining the intent.